collector/container/rhtap-slim.Dockerfile

```diff
@@ -0,0 +1,152 @@
+ARG BUILD_DIR=/build
+ARG CMAKE_BUILD_DIR=${BUILD_DIR}/cmake-build
+
+# TODO(ROX-20312): we can't pin image tag or digest because currently there's no mechanism to auto-update that.
+# TODO(ROX-20651): Use RHEL/ubi base image when entitlement is solved.
+# RPMs requiring entitlement: bpftool, cmake-3.18.2-9.el8, elfutils-libelf-devel, tbb-devel, c-ares-devel, jq-devel
```

Comment:
Just out of curiosity, why this particular cmake version is called out? As far as I recall 3.20 is actually what's being used currently.

Reply:
From https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/blob/rhacs-1.0-rhel-8/distgit/containers/rhacs-collector-slim/Dockerfile.in?ref_type=heads#L17